### PR TITLE
rename opendatakit to getodk.

### PR DESCRIPTION
**Pull request by issa-tseng**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+[ODK Central Backend](https://github.com/getodk/central-backend) is a RESTful API server that provides key functionality for creating and managing Open Data Kit data collection campaigns. It couples with [Central Frontend](https://github.com/getodk/central-frontend), an independent frontend interface, to form [ODK Central](https://github.com/getodk/central), a complete user-installable ODK server solution. While Central Frontend is the primary consumer of the ODK Central API, the API this server provides is fully public and generic: anything that can be done in the user interface can be done directly via the API.
```
Comment (matthew-white) [inline docs/api.md:4]: "Open Data Kit" here

File: README.md [hunk 2]
```diff
@@ -2,29 +2,29 @@
 
 ![Platform](https://img.shields.io/badge/platform-Node.js-blue.svg)
 [![License](https://img.shields.io/badge/license-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Build status](https://circleci.com/gh/opendatakit/central-backend.svg?style=shield)](https://circleci.com/gh/opendatakit/central-backend)
+[![Build status](https://circleci.com/gh/getodk/central-backend.svg?style=shield)](https://circleci.com/gh/getodk/central-backend)
 
-ODK Central Backend is a minimal [Open Data Kit](https://opendatakit.org/) server based on Node.js and Postgres. It is currently under development.
+ODK Central Backend is a minimal [Open Data Kit](https://getodk.org/) server based on Node.js and Postgres. It is currently under development.
```
Comment (matthew-white) [inline README.md:7]: "Open Data Kit" here

**Review by matthew-white — APPROVED** [review]
This is looking good to me. ✅ The changes here look right. I searched for remaining uses of "opendatakit", and the only places I saw had to do with OData or with XForm namespaces, which I believe aren't supposed to change. I also searched for:

^kit
[^a]kit
Kit
KIT

I saw two references to "Open Data Kit", which I think should be replaced with "ODK". Didn't see anything else though!

**Comment by yanokwa** [comment]
Agreed with the changes @matthew-white suggested.

**Pull request merged**